v1/mobileapps.yaml

```diff
@@ -270,54 +270,3 @@ paths:
                             from_backend.headers) }}'
               body: '{{from_backend.body}}'
       x-monitor: false
-
-  /mobile-text/{title}:
```

Comment:
We already tried to do that, and discovered there are clients using it, cf [T139029](https://phabricator.wikimedia.org/T139029), so we should be careful this time around.